src/components/mgt-person/mgt-person.ts

```diff
@@ -477,14 +478,17 @@ export class MgtPerson extends MgtTemplatedComponent {
       // in some cases we might only have name or email, but need to find the image
       // use @ for the image value to search for an image
       if (this.personImage === '@' && !this.personDetails.personImage) {
-        this.loadImage();
+        const image = await getPersonImage(graph, this.personDetails);
+        if (image) {
+          this.personDetails.personImage = image;
+          this.personImage = image;
+        }
       }
       return;
     }
 
     // Use userId or 'me' query to get the person and image
     if (this.userId || this.personQuery === 'me') {
```

Comment:
Make this else if and remove return from above

Reply:
I am going to fold in the other pr #384 and reflect all the changes so feel free to close that one later